util/test/check_annotations.py

```diff
@@ -137,9 +137,9 @@ def check_pr_number_dates(ann_data):
                     pr_num = re.match(r'.*#(\d+)', text).group(1)
                     if pr_num in pr_to_date_dict:
                         pr_date = pr_to_date_dict[pr_num]
-                        if pr_date == date:
+                        if pr_date >= date:
                             warnings.warn('Warning: annotation date for "{0}: '
-                                          '{1}" appears to be the same as the '
+                                          '{1}" appears to be earlier than the '
```

Comment:
earlier than or the same as?

Reply:
Right. Also changed "appears to be" -> "is" to make it sound more like we know what we are doing.